build-winpe/build-razor-winpe.ps1

```diff
@@ -113,7 +112,7 @@ copy-item $wim $output
 $wim = join-path $output "winpe.wim"
 
 
-
+$env:Path = ($env:Path + ";$adk\..\..\Deployment Tools\amd64\DISM")
```

Comment:
Since it's already in quotes and being assigned to the `env:Path` variable, it seems to be working as-is without any escaping.